examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs

```diff
@@ -1012,6 +1014,13 @@ public string Next()
 
                 // Unset for the root listing group filter node.
```

Comment:
nit: I know this comment's not changing, but I find it a little confusing. Perhaps a bit more explanation would help, like "Since this is the root node, do not set the ParentListingGroupFilter. For all other nodes..."